src/ornament.ts

```diff
@@ -2,29 +2,22 @@
 // @author Cyril Silverman
 // MIT License
 
+import { getBottomY, getInitialOffset, getTopY } from './articulation';
 import { Element } from './element';
-import { Glyph } from './glyph';
-import { Modifier } from './modifier';
+import { Modifier, ModifierPosition } from './modifier';
 import { ModifierContextState } from './modifiercontext';
-import { Stem } from './stem';
+import { Note } from './note';
 import { StemmableNote } from './stemmablenote';
 import { Tables } from './tables';
 import { TickContext } from './tickcontext';
-import { Category, isTabNote } from './typeguard';
-import { defined, log, RuntimeError } from './util';
+import { Category } from './typeguard';
+import { log } from './util';
 
 // eslint-disable-next-line
 function L(...args: any[]) {
   if (Ornament.DEBUG) log('Vex.Flow.Ornament', args);
 }
 
-export interface OrnamentMetrics {
-  xOffset: number;
-  yOffset: number;
-  stemUpYOffset: number;
-  reportedWidth: number;
-}
-
```

Comment:
I like the way we have gone, with just using SMuFL glyphs directly, but perhaps there is some value to being able to hand tune the position of glyphs. Maybe later on we'll need to reintroduce little customizations like the xOffset / yOffset.